.github/workflows/validate.yml

```
@@ -16,7 +16,7 @@ jobs:
     if: ${{ !contains(github.head_ref, 'all-contributors') }}
     strategy:
       matrix:
-        node: [10.13, 10, 12, 14, 16]
+        node: [12, 14, 16]
```

Is there be a minimum version of v12 we should be testing against?

---

`12.13` was first LTS version

https://nodejs.org/en/blog/release/v12.13.0

```suggestion
        node: [12.13, 12, 14, 16]
```

---

But we can also check against Node 12.0 if we want

```suggestion
        node: ['12.0', 12, 14, 16]
```